docs/concepts/toc.yml

```diff
@@ -15,10 +15,24 @@
     href: ../basemap-gallery.md
   - name: Bookmarks View
     href: ../bookmarks-view.md
+  - name: Compass
+    href: ../compass.md
+  - name: FeatureDataField
+    href: ../feature-data-field.md
+  - name: Legend
+    href: ../legend.md
+  - name: Measure Toolbar
+    href: ../measure-toolbar.md
```

Comment:
Would it make more sense to put all these in conceptual docs folder instead, since that's what they are?

Reply:
I think it would, but I didn't want to make that change as part of this PR.

Reply:
Aaah you're right that's where the other pages are too.